ecdh_test.go

```diff
@@ -0,0 +1,147 @@
+package secp256k1_test
+
+import (
+	"encoding/hex"
+	"reflect"
+	"testing"
+	"vulpemventures/go-secp256k1-zkp"
+)
+
+func TestEcdh(t *testing.T) {
+	ctx, err := secp256k1.ContextCreate(secp256k1.ContextSign | secp256k1.ContextVerify)
+	if err != nil {
+		panic(err)
+	}
+
+	type args struct {
+		ctx     *secp256k1.Context
+		pubKey  string
+		privKey string
+	}
+	tests := []struct {
```

Comment:
Are these tests taken from the js version?
Anyway, it should be fine to import the [`ecdh.json`](https://github.com/vulpemventures/secp256k1-zkp/blob/master/test/fixtures/ecdh.json) fixtures file from the json version and use it as the source of our test vectors.